### PR TITLE
IK-924: Added a section on signed urls containing diacritics

### PR DESCRIPTION
**Pull request by hrhv**

### CODE REVIEW

File: features/security/signed-urls.md [hunk 1]
```diff
@@ -148,3 +148,35 @@ var signature = crypto.createHmac('sha1', "your_private_key").update(str).digest
...
+To ensure that your signed URLs that contain a diacritic character works, you would need to use an encoded string at the time of signature generation.
+
+```javascript
+/**
+ * To encode a URL like "https://ik.imagekit.io/<imagekit_id>/default-image-with-é.jpg",
+ * make sure you use the UTF-8 encoded string corresponding to the diacritic character.
+ * In this case, we need to use "https://ik.imagekit.io/<imagekit_id>/default-image-with-e%CC%81.jpg"
+ **/
+var imageURL = imagekit.url({
+    path : "/default-image-with-e%CC%81.jpg",
```
Comment (imagekitio) [inline features/security/signed-urls.md]: shouldn't we replace this with `encodeURIComponent(...)`?

File: features/security/signed-urls.md [hunk 2]
```diff
@@ -148,3 +148,35 @@ var signature = crypto.createHmac('sha1', "your_private_key").update(str).digest
...
+
+Most browser engines encode the diacritics when used within a URL. For example,
+
+```
+é -> e%CC%81
+
+and so on...
+```
+
+To ensure that your signed URLs that contain a diacritic character works, you would need to use an encoded string at the time of signature generation.
```
Comment (imagekitio) [inline features/security/signed-urls.md]: Be explicit in conveying that signed URL should be of encoded string.

**Review by imagekitio — CHANGES_REQUESTED** [review]

File: features/security/signed-urls.md (hunk 1 above)
Comment (hrhv) [reply features/security/signed-urls.md]: Makes sense! Will update this.

File: features/security/signed-urls.md (hunk 2 above)
Comment (hrhv) [reply features/security/signed-urls.md]: ACK

File: features/security/signed-urls.md (hunk 1 above)
Comment (hrhv) [reply features/security/signed-urls.md]: Done

File: features/security/signed-urls.md (hunk 2 above)
Comment (hrhv) [reply features/security/signed-urls.md]: Done

**Pull request merged**